src/components/icons/Etherscan.tsx

```diff
@@ -0,0 +1,17 @@
+import { ThemeContext } from 'contexts/Theme/ThemeContext'
+import { useContext } from 'react'
+
+export const Etherscan = ({ size }: { size?: number }) => {
+  const { forThemeOption } = useContext(ThemeContext)
+  return (
+    <img
+      src={forThemeOption?.({
+        dark: '/assets/icons/etherscan_od.svg',
+        light: '/assets/icons/etherscan_ol.svg',
+      })}
+      alt="Etherscan logo"
+      width={size || 15}
+      height={size || 15}
+    />
```

Comment:
**suggestion** should be able to use `next/image` for this

Reply:
i am against using next image. It is a headache and the consensus from others is that its not worth using in the version of next we use. 

Apparently it's better in later versions